lib/rubocop/cop/lint/float_comparison.rb

```diff
@@ -15,6 +15,10 @@ module Lint
       #   x == 0.1
       #   x != 0.1
       #
+      #  # good - comparing against zero
+      #   x == 0.0
+      #   x != 0.0
```

Comment:
Nits: Can you swap the positions of the `# good - using BigDecimal` example? It's to contrast it with the bad case, as `# good - using BigDecimal` is more illustrative when placed opposite.